README.md

```diff
@@ -31,13 +29,253 @@ _NB: There is no compatibility with Windows environments at this time._
+
+# Expected Output:
+#   docker compose -p issuer -f /Users/username/path/to/sh-id-platform/infrastructure/local/docker-compose-infra.yml up -d redis postgres vault
+#   [+] Running 4/4
+#   ⠿ Network issuer-network       Created                                                                                   0.0s
+#   ⠿ Container issuer-vault-1     Started                                                                                   0.5s
+#   ⠿ Container issuer-redis-1     Started                                                                                   0.4s
+#   ⠿ Container issuer-postgres-1  Started  
+````
 
-### Option 2 - Standalone mode
+If you want to remove all the services (ignore the warnings):
+
+```bash
+# FROM: ./
+
+make down; 
+# (Equivalent)
+#   docker compose -p issuer -f ./infrastructure/local/docker-compose-infra.yml down --remove-orphans -v;
+
+# Expected Output:
+#   docker compose -p issuer -f /Users/username/path/to/sh-id-platform/infrastructure/local/docker-compose-infra.yml down --remove-orphans
+#   [+] Running 4/3
+#   ⠿ Container issuer-postgres-1  Removed                                                                                   0.2s
+#   ⠿ Container issuer-redis-1     Removed                                                                                   0.2s
+#   ⠿ Container issuer-vault-1     Removed                                                                                   0.2s
+#   ⠿ Network issuer-network       Removed                                                                                   0.0s
+#   docker compose -p issuer -f /Users/username/path/to/sh-id-platform/infrastructure/local/docker-compose.yml down --remove-orphans
+#   WARN[0000] The "DOCKER_FILE" variable is not set. Defaulting to a blank string. 
+#   WARN[0000] The "DOCKER_FILE" variable is not set. Defaulting to a blank string. 
+#   WARN[0000] The "DOCKER_FILE" variable is not set. Defaulting to a blank string. 
+#   WARN[0000] The "DOCKER_FILE" variable is not set. Defaulting to a blank string.
+```
+
+#### Import Wallet Private Key To Vault
+
+In order to secure our wallet private key so that the issuer can use it to issue claims/credentials, we'll store it in the hashicorp vault.
+
+ℹ️ **NOTE:** Make sure the wallet you're providing has Testnet Matic to send transactions.
+
+```bash
+# FROM: ./
+
+# Make sure to verify that the issuer-vault-1 is full initialised to avoid: "Error writing data to iden3/import/pbkey: Error making API request."
+make private_key=YOUR_WALLET_PRIVATE_KEY add-private-key;
+# (Equivalent)
+#   docker exec issuer-vault-1 vault write iden3/import/pbkey key_type=ethereum private_key=YOUR_WALLET_PRIVATE_KEY;
+
+# Expected Output:
+#   docker exec issuer-vault-1 \
+#           vault write iden3/import/pbkey key_type=ethereum private_key=YOUR_WALLET_PRIVATE_KEY
+#   Success! Data written to: iden3/import/pbkey
+```
+
+#### **Add Vault To Configuration File**
```

Comment:
Bold is not needed, this is already a header.

Reply:
Done.